cpp/harvest/nacjd.cc

```diff
@@ -61,7 +61,9 @@ namespace {
         "\n"
         "convert_JSON_to_MARC json_path marc_path\n"
         "\t- json_path: The input JSON file.\n"
-        "\t- marc_path: The output MARC file.\n");
+        "\t- marc_path: The output MARC file.\n"
+        "get_statistics json_path\n"
```

Comment:
Please don't forget to add an empty line between this mode and the previous one.